frontend/timesheet/timesheetController.js

```diff
@@ -1155,6 +1200,52 @@ angular.module('mifortTimesheet.timesheet', ['ngRoute', 'constants'])
                 });
             };
 
+            $scope.saveAllHours = function () {
+                //Notification({message: 'Button clicked!', delay: 5000}, 'warning');
+
+                const dailyHours = $scope.filteredLogs;
+
+                let status = 0;
+                dailyHours.forEach((dailyHour) => {
+                    if (dailyHour.time > 8 && dailyHour.time <= 16) {
+                        if(!status) {
+                            status = 1;
+                        }
+                    }
+                    if (dailyHour.time > 16 && dailyHour.time <= 24) {
+                        if(status < 2) {
+                            status = 2;
+                        }
+                    }
+                    if (dailyHour.time > 24) {
+                        status = 3;
+                        return;
+                    }
+                })
+
+                if(status === 1) {
+                    Notification({
+                        message: 'You have filled in more then 8h per day',
+                        delay: 4000
```

Comment:
If you delay the notification, and someone enters first 17 hours and then changes to 7, he will still see that warning, because a new value won't cancel that notification.

I would say just drop that >8 and >16 warnings and leave >24 error with no save.